helm/nginx-ingress-controller-app/templates/controller-deployment.yaml

```diff
@@ -11,9 +11,13 @@ metadata:
     prometheus.io/port: '{{ .Values.controller.metrics.port }}'
     prometheus.io/scrape: 'true'
 spec:
-  {{- if and ( (not .Values.controller.autoscaling.enabled) (or (not (index .Values.configmap "hpa-enabled")) (ne (index .Values.configmap "hpa-enabled") "true") ) ) }}
```

Comment:
```panic: template: nginx-ingress-controller-app/templates/controller-deployment.yaml:15:59: executing "nginx-ingress-controller-app/templates/controller-deployment.yaml" at <ne (index .Values.configmap "hpa-enabled") "true">: error calling ne: incompatible types for comparison
```
To mitigate parsing error from helmclient 1.0.2, I need to rephrase it like below.